api/src/services/files.ts

```diff
@@ -107,12 +165,31 @@ export class FilesService extends ItemsService {
 			const stream = await storage.location(data.storage).read(payload.filename_disk);
 			const { height, width, description, title, tags, metadata } = await this.getMetadata(stream);
 
-			payload.height ??= height ?? null;
-			payload.width ??= width ?? null;
-			payload.description ??= description ?? null;
-			payload.title ??= title ?? null;
-			payload.tags ??= tags ?? null;
-			payload.metadata ??= metadata ?? null;
+			if (!payload.height && height) {
```

Comment:
These should overwrite no matter what right?

Reply:
This overwrites the height if the user does not supply the height in the upload, and there is a new height. If the user supplies their own height we ignore, and if there is no new height detected then it won't set the field to null.

Are you saying that it _should_ set the field to null?